### ISSUE TITLE
Boutiques integrator should support shells other than bash

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#779]
A boutiques descriptor can provide a property that says the command to run is from a specific interpreter, e.g.

```json
{
  "command-line": "blah blah |& whoopsie",
  "shell": "tcsh"
}
```

In CBRAIN we run the command under `bash`, always, which can be a problem. In the example above, `|&` is a csh piping that redirects both stdout and stderr, while in bash this will only cause a bash syntax error.

**Comment by MontrealSergiy** [comment]
I think integrating bosh is big enough to be considered as a full fledged feature (add full boutiques support) rather than a bugfix

**Comment by prioux** [comment]
I'm thinking of adding just two line around the generated command:

```bash
theshell <<'QUOTED_COMMAND'
normal_command here
QUOTED_COMMAND
```

there theshell is what the descriptor specifies. I would add these two lines only if theshell is not already 'bash'.

**Comment by prioux** [comment]
Fixed by #815
